Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "avail-light"
-version = "0.1.0"
+version = "2.0.0"
 edition = "2021"
 authors = ["Polygon Avail Team"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = { git = "https://github.com/maticnetwork/avail.git", rev = "76e2b45" }
-kate-proof = { git = "https://github.com/maticnetwork/avail.git", rev = "76e2b45" }
+kate-recovery = { git = "https://github.com/maticnetwork/avail.git", rev = "062d946" }
```

Comment:
Can we use `tag` instead of `rev`? Looks a bit cleaner IMO.

Reply:
+1 to use tags instead of `rev`.